backend/utils.test.js

```diff
@@ -51,4 +51,10 @@ describe('getTime()', () => {
   it('should return Invalid message if no date time string is used', () => {
     expect(getTime()).toEqual('Invalid date or time')
   })
+  it('should trim all fields of an object', () => {
```

Comment:
you've added this to the `getTime` describe block, should really get its own
```
describe('trimObjValues()', () => {
```
deescribe block